SPECS/libsolv/libsolv.spec

```diff
@@ -10,6 +10,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 BuildRequires:  cmake
 BuildRequires:  rpm-devel
+BuildRequires:  libzstd-devel
```

Comment:
Does this increase the size of the base image? If so, by how much?

Reply:
Good question! I still need to build the artifacts to confirm, but `rpm` already has a runtime `Requires` on `libzstd.so.1()(64bit)` and that's exactly what `libsolv` ends up requiring too with this change. It seems safe to say that anywhere `libsolv` is meaningfully used in our system that `rpm` would be expected to be there too, so I would not expect to see an image size increase.